tools/precommit.py

```diff
@@ -140,9 +139,18 @@ def setup_tizenrt_repo(tizenrt_root):
         ex.check_run_cmd('git', ['clone',
             'https://github.com/Samsung/TizenRT.git',
             tizenrt_root])
+    # The following two do not have to succeed
+    ex.run_cmd('git', ['--git-dir', tizenrt_root + '/.git/',
+                       '--work-tree', tizenrt_root,
+                       'checkout', 'master'])
+    ex.run_cmd('git', ['--git-dir', tizenrt_root + '/.git/',
+                       '--work-tree', tizenrt_root,
+                       'branch', '-D', 'iotjs_build'])
+    # These have to however
     ex.check_run_cmd('git', ['--git-dir', tizenrt_root + '/.git/',
                              '--work-tree', tizenrt_root,
-                             'checkout', TIZENRT_COMMIT])
+                             'checkout', '-b', 'iotjs_build', TIZENRT_COMMIT])
```

Comment:
`iotjs_build` is the branch that was removed.

Reply:
Yes, so it can be re-created with (perhaps new) revision specified.